device_attributes/index.bs

```diff
@@ -0,0 +1,180 @@
+<pre class='metadata'>
+Title: Device A API
+Shortname: managed-configuration
+Level: 1
+Editor: Anqing Zhao 130190, Google LLC https://www.google.com, anqing@google.com
+Status: w3c/CG-DRAFT
+URL: https://wicg.github.io/WebApiDevice/device_attributes
+Abstract: This document defines a web platform API that enables developers to query device information (device ID, serial number, location, etc) from managed devices. This API is invaluable for such use cases as Virtual Desktop Infrastructure (VDI) and Context-based Configuration.
+Repository: https://github.com/WICG/WebApiDevice
+Group: wicg
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+spec:url; type:dfn; text:origin
+spec:html; type:dfn; for:/; text:global object
+spec:webidl; type:dfn; text:resolve
+spec:webidl; type:dfn; text:record
+</pre>
+
+<pre class="anchors">
+
+</pre>
+
+# Model # {#model}
+
+## Managed devices ## {#managed-devices}
+
+The API is presumed to be used on devices which are not fully controlled by the end user, but rather by an external entity, the device administrator. Device administrators are given the power to fully control the managed devices.
+
+## Managed web applications ## {#managed-web-applications}
+
+The API is presumed to be used by web applications, which are understood and managed by the device administrators.
+
+## Device attributes ## {#device-attributes}
+
+The API is presumed to be used with a management infrastructure which allows administrators to manage a fleet of devices and configure properties for those devices.
+
+### Annotated Asset ID ### {#annotated-asset-id}
```

Comment:
Create a definition so that it can be linked as regular text below.

```suggestion
### <dfn>Annotated Asset ID</dfn> ### {#annotated-asset-id}
```